README.md

```diff
@@ -21,7 +21,9 @@ There are several steps that need to be done when rebasing a package. The goal o
 - new sources are extracted and added as a remote repository
 - `git-rebase` is used to rebase the commits on top of new sources
 - original patches are modified/deleted accordingly
-- changes are reflected in the spec file
+- rebased sources are stored in *rebase-helper-results/rebased_sources*. These files are needed for the new version.
+- Patch *changes.patch* containing changes between original and rebased spec file and patches is located in *rebase-helper-results*
```

Comment:
I think this sentence deserves some better wording. I'd use `File` instead of `Patch` to avoid redundancy at least. But I think it should be rewritten completely.